inspec/a2-api-integration/controls/iam/iam_apis.rb

```diff
@@ -1227,4 +1031,282 @@
       end
     end
   end
+
+  describe 'policies API' do
+
+    before(:all) do
+      resp = automate_api_request("/apis/iam/v2/roles",
+        http_method: 'POST',
+        request_body: {
+          id: ROLE_ID,
+          name: "display name !#$#",
+          actions: ["test:some:action", "test:other:action"]
+        }.to_json
+      )
+      expect(resp.http_status).to eq 200
+
+      resp = automate_api_request("/apis/iam/v2/policies",
+        http_method: 'POST',
+        request_body: {
+          id: POLICY_ID,
+          name: 'gotta catch em all',
+          members: ["user:local:member"],
+          statements: [
+            {
+              effect: "DENY",
+              role: ROLE_ID,
+              projects: ["*"]
+            },
+            {
+              effect: "ALLOW",
+              actions: ["test:some:action"],
+              projects: ["*"]
+            }
+          ]
+        }.to_json()
+      )
+      expect(resp.http_status).to eq 200
+    end
+
+    after(:all) do
+      resp = automate_api_request("/apis/iam/v2/policies/#{POLICY_ID}", http_method: 'DELETE')
+      expect(resp.http_status).to eq 200
+
+      resp = automate_api_request("/apis/iam/v2/roles/#{ROLE_ID}", http_method: 'DELETE')
+      expect(resp.http_status).to eq 200
+    end
+
+    describe 'list default policies' do 
+      it 'includes the IAM v2 default policies' do
+        resp = automate_api_request('/apis/iam/v2/policies')
+        expect(resp.http_status).to eq 200
+  
+        policy_ids = resp.parsed_response_body[:policies].map { |u| u[:id] }
+        expect(policy_ids).to include("administrator-access")
+        expect(policy_ids).to include("ingest-access")
+        expect(policy_ids).to include("editor-access")
+        expect(policy_ids).to include("viewer-access")
+      end
+  
+      it 'the editors default policy includes editor role' do
+        resp = automate_api_request('/apis/iam/v2/policies')
+        expect(resp.http_status).to eq 200
+  
+        all_policies = resp.parsed_response_body[:policies]
+        policies = all_policies.select{ |p| /^team:local:viewers$/.match(p[:members][0]) }
+        expect(policies.length).to eq 1
+      end
+  
+      it 'the viewers default policy includes editor role' do
+        resp = automate_api_request('/apis/iam/v2/policies')
+        expect(resp.http_status).to eq 200
+  
+        all_policies = resp.parsed_response_body[:policies]
+        policies = all_policies.select{ |p| /^team:local:editors$/.match(p[:members][0]) }
+        expect(policies.length).to eq 1
```

Comment:
```suggestion
      it 'the viewers default policy includes viewer user' do
        resp = automate_api_request('/apis/iam/v2/policies')
        expect(resp.http_status).to eq 200
  
        all_policies = resp.parsed_response_body[:policies]
        policies = all_policies.select{ |p| /^team:local:viewer$/.match(p[:members][0]) }
        expect(policies.length).to eq 1
```

Reply:
Good catch-- a similar goof was in the test above, too